packages/manager/modules/pci/src/projects/project/streams/stream/subscriptions/resetCursor/index.js

```diff
@@ -0,0 +1,24 @@
+import angular from 'angular';
+import '@uirouter/angularjs';
+import 'oclazyload';
+
+const moduleName = 'ovhManagerPciStreamsStreamSubscriptionsResetCursorLazyLoading';
+
+angular
+  .module(moduleName, [
+    'ui.router',
+    'oc.lazyLoad',
+  ])
+  .config(/* @ngInject */($stateProvider) => {
+    $stateProvider.state('pci.projects.project.streams.stream.subscriptions.resetCursor.**', {
+      url: '/cursor/reset',
```

Comment:
Shouldn't be `/reset-cursor`? (regarding my comment about the following state `pci.projects.project.streams.stream.backlogRetention`)?